examples/config.sample

```diff
@@ -53,6 +53,12 @@
 #       most verbose.
 #       Default:<info>
 #
+#   FcuId
```

Comment:
suggestion: LogSystemId, and move it near to "LogMode".

Reply:
ok